test/fixtures/build.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
```

Comment:
Is this fixture in use? You may need to `chmod +x` this file to executable.

Reply:
It is and I did, I believe :)

Reply:
My bad. It's already executable. The GitHub changes didn't show its file mode.